driver-core/src/test/unit/com/mongodb/connection/ServerSelectionSelectionTest.java

```diff
@@ -33,6 +33,7 @@
 import org.bson.BsonString;
 import org.bson.BsonValue;
 import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
```

Comment:
We seem to be importing jetbrains annotations (via intellij) and then reinventing them as well. eg VisibleForTesting.

Reply:
Importing from `org.jetbrains.annotations` is my mistake, thanks for spotting!

As for `VisibleForTesting`, I actually reinvented the annotation from the Android API, see https://developer.android.com/reference/androidx/annotation/VisibleForTesting. As far as I know, the authors are the only ones who came up with the `VisibleForTesting.otherwise` idea. But originally, I was it was Guava that introduce the annotation: https://guava.dev/releases/snapshot-jre/api/docs/com/google/common/annotations/VisibleForTesting.html.

I do not see another way to get the `VisibleForTesting` annotation in our code without introducing a dependency on Guava/Jetbrains/Android. And having this annotation is definitely better than writing comments like _"Is package-access for the purpose of testing and must not be used for any other purpose outside of this class"_, which I was doing previously.

Reply:
:+1: